packages/cerebral/typings/index.d.ts

```diff
@@ -0,0 +1,109 @@
+// <reference types="node" />
+// <reference types="function-tree" />
+
+
+declare module 'cerebral/devtools' {
```

Comment:
Nope. Can't find a way to handle `'cerebral/devtools'` properly. Suggestions welcome.